presto-spi/src/main/java/com/facebook/presto/spi/statistics/PlanStatistics.java

```diff
@@ -29,6 +36,7 @@ public static PlanStatistics empty()
         return EMPTY;
     }
 
+    @ThriftConstructor
```

Comment:
nit: requireNonNull(outputSize, "rowCount can not be null");

the error message should outputSize is nulll